test/random-battles/all-gens.js

```diff
@@ -372,7 +380,7 @@ describe('[Gen 9] BSS Factory data should be valid (slow)', () => {
 		const genNum = 9;
 
 		for (const speciesid in setsJSON) {
-			const vType = 'battlestadiumsingles';
+			const vType = 'bssfactory';
```

Comment:
I believe this should be `bssregh`, since it's validating off of the base battle stadium format legality. Obviously a fix of checking if `gen9bssregh` exists would make sure people update this since there's no easy way to automate this apart from trying to assign an alias to the BSS format that's active in `aliases.ts`, but if you can figure something else out I'm all ears.

Reply:
All I could come up with so far is validating that the formats exist. The alias is an interesting idea, kind of like how 'base' is an alias for the current gen's dex.